.gitignore

```diff
@@ -25,6 +25,10 @@ python/nvfuser_common/cmake
 python/nvfuser/*.so
 python/nvfuser_direct/*.so
 
+# Symbolic links generated by CMake
+nvfuser
+nvfuser_direct
```

Comment:
this looks a bit dangerous to me, what about `/nvfuser` and `/nvfuser_direct` to only ignore those in the root path?